src/test/resources/hudson/security/planetexpressExtGroups_dnWithoutCn.ldif

```diff
@@ -0,0 +1,2506 @@
+ 1/7gvH3Bf+QRwzPaZ7RUiXCArohNI9BqDqPp4dL6PaKcJ5Mw6yFPhYAZytqi2hdGhm2gW14f3Kd
+ pWbf6XDpTMrV+hTT5te+AIpF8jmi7htpYFHIGwf/lo3DMaT37jCbP6sx+2BeoFSujcCoFwinpuJ
+ EzGpVwLWJxSveUI44+hcw4QfliuWgx9Ai5PEuVRUinauBZraHDZZYLbUAkELE0//ADuXBHtofup
+ d6wTwu/cBxACa+mId33EXt+o9jCCA6JFpl31qDb1lccqi+OgrzOA6FfQ3HY5WsC8heBSbElwoX3
+ KJkuXMC24sVsDxPDkTSOREfxSiCg0bY4elHO8pFYdnD2Zd43f+manH7tNPmDOpNIS7NZBFwQZ+k
+ v8AIg/vs9VeRfMHXxGAAfRBvCPzBxXJ+pT3/c+SVW/6h6mO31GvMo7/AKnki1FuLHDfSsR9ptmD
+ UcXeZIvqK/8AmEOmYful8HELGq3K3gwFhEq4jjrVIoFaAAxdWv5Vl6ZQNEOMui8G6FX5JVD6DwB
+ Rwj0sOJWIPQY+8xJq97XfmJt/EH0WFpwlGdk23lS6/wAO4Jj7YUdF/wCF/wBf4ORfsUgd0G26Dp
+ PcQC4SJx+mUHMy5nwfU+MT0ZR3fqJ5GLUW49CV0aR8QR1roIg9wYpyRUV8sKfEcpk+TAfRB3Knv
+ WUN9z+Baw/HDdlRUeEMP5v3SNoPddPlSBuVthdgB6QvmI6bsxf9aWfEF4BsqbxsXgD+B80hdi0B
+ hJdNtnKVLO2kG2VynloNAS7TXuD4zBHEqvEp5z8RDkPiJTFkbc5l40fMEUW4+ZVwECvBH26H0di
+ KKKX5eId8wGfMyHuJyYB5taOXB9nEJA+iftM+fyBJnEXXIStHCrVgArubOj92fIQ8QwOYBrywLG
+ KNmUIEzWJ9gU8/wAQQRwjzGqy29NuAru7PeDTkqAIzWUFuizKtTCTVfjxb6DMfbxfwa2XpFEdkd
+ LltBVdtIBYuDMq9hAPUccNy3n9wb2ETyIDpl/cyiVFjoVwLjbovtAigzH1C0VQbg5oh2Q/SLR8z
+ MYVSUw352FvGz0fnf0dht3+YAPUvU06sO82D7G5gD3Q7B2I7I34SRBAAFSvdV5/ip/g1/jsZ7wR
+ LeKCK12WihZlLUZYOGpvYQrvXuBjB9SvMSOIiITcVNfZLO2/cvzqI7QDzOX8BuDUGIPcMswg+sI
+ KIqL5YU9Ta98Th9w/pEVoOxhdt/rX8C2h53TJY8k+B/FHgAYHjv4zMz4wgHKKGviKFCnzMc+Bgd
+ 4CAOkTZ+NM3FVC3qGAFnDsVbHyw73ZXvkLNiATnUCooM59zLx6i+RjPJF+ZbH5RhtGFqKeopBuU
+ lDF9T9IVnN9QF4IXzFh5mBXYjZ+KnJ7EXtAD2K94Pn+A3T9pVnkkA8eA2D/qmOrvUdzoyjvWo3Z
+ hYWXgLbAIvUa2ragAAoMARGeGvdSCryUxNPldpbewatRasBU1CwMlcdRSHQg6QHaBjMCBZlWgyV
+ S3B+CxL4UXeFwnBSRb2yL2wSMuQoWhqmeK1m2oB07wcuauzacgcHNrmoiZhC1BiHmKkW9z5jfRu
+ DFFjHvCdw4cQYrxHxxBo8Qza9OCoN4JYexmWVfOYWz3bmv+uZQKztn01fHTyDWM36E/MARkCxHY
+ xta/FoLXpxeulgGBF0x5O8ZypmKSlgFrKDblPdxQB+IpZiPSwXu26WhKOMFkRQAEhK79UL8Jqdx
+ 55HY5Mxxtu9SHYtrO0QQbEsTmdwzLZ9Tem/Exc9S9UqJccRS+jToqYSrg+YJmvjtNrmK3cQF7z5
+ GErS8QoHj/ALmz2Tbh8sSPCKRvANsT5bQIYsP4G1oNwrX99An4hbCzsVC3dU1+BjQ3HKz8JeQXb
+ CmBw4GWsDCIiJsZewjy7X+pSjK2bGo8HWlKqiGQpDgN5VTdQeDGloGO1QThDxCTmIV245aLfKMt
+ gpd+4qGcnmYfEWFxIy6ixRWxairjSMDxAmSp9HeF5dYl3L4nZudtxo1sCo4Duxkp+UvPj/mVpOr
+ mZq5u/SKCAWUHCcIcBsThHqWuuTq0JMscrraVmAc5Bp7Nk+1RwF0gZuzo4tXiWaN1RdryH08KQN
+ I800Bx2vpSyCQdrfyX+FxGKTRUbgKC0UpS36hFrZlzVstwHRmFImyf0nY/9Yr4/wCxODzPBd8Qg
+ HUF5gACovgFqDFYKL3Bq+VgOl3FEw+JSafqK9n1G+Rv3iDoWO5hNItS4FweJXBKMRUdA0XDEVWz
+ meMwdry5lJ8IBHiXe5/xExMJ5I+ud7fW86stfBgg94XK5RkORB6IkVRaWZ4S9bo7QCs0VUmKhRD
+ lbZCKTfXDhKhlyqy6mSFVJZ1MAX3AV0kBt1Dgd6SkThs6GXGd0gInfIcaF+WHBQV2R80A5QRwlv
+ qjmxGjN4xaRdlbp3dwT6REsR6pYOgvGFtXDGi3QzfnFRXPcPlwU1DO4lGItwYUIhiYxFaddmNPj
+ obRXFNuigXB03DOeJcd4gv1OCYAfMME5WXZCOy4VD3KfTc0/wC95keX+UV/aU+DB3zeGXnYqijP
+ 5ioKdIQeCzEHjTmpwPKkWd8QsUaXVYNWveDwmBoFUUwaJaYSxLcEG0lAHe2+JfsZ40AQFqkW7OO
+ YGVIDkYBa28uAWJs+9wBptnCeksRfwMJlGvOlWF3UIaRJtjUCgOxHYxkRKA4s0aL2QaluFBQbES
+ Hl0B2Pf4rB7SCLBsFZa8AlYfIQkVIkPvouYj1LlvRdRrLHxHufcBdREEa6FSELRw41BuXR5iQmw
+ xtSuWZ07IsQdfcbAgX4f5Yck/sTC77kGk+pYE1tu0v0sQi9bGFztWvmmFN669ZzsDbvW1/JmtYQ
+ TIiO/SturEJY
```

Comment:
Can you please change these names to something random/anonymous? Seems better to not use the actual names of groups from a real company's LDAP schema.

Reply:
@dwnusbaum I don't get this point :( Can you clarify it better? Thanks

Reply:
IIRC, `C3ALM`, `ccc_c3alm-support-developer`, `ccc_c3alm-support-product-owner`, etc. were real names from an actual company's LDAP schema that came up in relation to this ticket, but maybe I am misremembering.

Reply:
Good point, it has been taking into account